bin/etc-update

```diff
@@ -32,13 +32,13 @@ get_config() {
 		"${PORTAGE_CONFIGROOT}"etc/etc-update.conf)
 }
 
-OS_RELEASE_ID=$(cat /etc/os-release 2>/dev/null | grep '^ID=' | cut -d'=' -f2 | sed -e 's/"//g')
+OS_RELEASE_POSSIBLE_IDS=$(source /etc/os-release >/dev/null 2>&1; echo ":${ID}:${ID_LIKE}:")
```

Comment:
I originally overlooked mentioning, that ID_LIKE can contain multiple, space-separated IDs. So this would really need to be `${ID_LIKE/ /:}` or it would not detect any ID_LIKE that has multiple associations.

Reply:
Good point, then `${ID_LIKE/ /:}` will only separate the first 2 IDs with `:`, what about if there are 3 of them?
I think we want `${A_LIKE// /:}` here, do you agree?

Reply:
…or you could just, like, use space as the separator and then you wouldn't have to replace anything.

Reply:
And then also quote the part in the case statement, then be personally responsible for making sure no one ever accidentally tries to optimize the leading and trailing spaces out of OS_RELEASE_POSSIBLE_IDS. But sure.

I'd personally avoid using spaces as a global pattern match delimiter.

Reply:
With spaces, the case patterns become quite heavy and more prone to errors.